python/setup.py

```diff
@@ -120,6 +120,7 @@
 # cumlcomms and nccl are still needed for multigpu algos not based
 # on libcumlprims
 libs = ['cuda',
+        'cudart',
```

Comment:
what pushed for this addition?

Reply:
same question

Reply:
It's not directly related to the PR itself, but it seemed like it fixed some problem on the Python side that appeared under rare circumstances. I'll remove it for now.